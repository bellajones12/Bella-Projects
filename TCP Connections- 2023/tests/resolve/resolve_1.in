com
google.com
www.google.com

