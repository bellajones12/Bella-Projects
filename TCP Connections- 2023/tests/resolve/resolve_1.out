1234
8888
1568
